bedrock/firefox/templates/firefox/challenge-the-default/includes/heroes/hero-v4.html

```diff
@@ -0,0 +1,19 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<!-- New Device -->
+<div class="c-hero-text">
```

Comment:
Doh, that's a regression. Because it's using flexbox it collapses to the content width and when I reduced the size of the text the whole box shrank. Fixed now.